educator.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'person'
+require_relative 'high_five'
+
+class Educator < Person
+  attr_reader :salary,:target_raise
+
+  def initialize(options={})
+    super
+    @phase = 3
```

Comment:
This variable shows up in a few classes. Could you set it as a constant? ```PHASE = 3 ``` ? How could that be implemented?